packages/rsocket-router/src/router/ReactiveSocketRouter.ts

```diff
@@ -60,11 +56,16 @@ export class ReactiveSocketRouter<C> {
       acceptor: {
         accept: async (payload) => {
           const { max_concurrent_connections } = this.options ?? {};
-          if (max_concurrent_connections && this.activeConnections >= max_concurrent_connections) {
-            throw new errors.JourneyError({
-              code: '429',
+          // wss.clients.size includes this connection, so we check for greater than
+          // TODO: Share connection limit between this and http stream connections
+          if (max_concurrent_connections && wss.clients.size > max_concurrent_connections) {
```

Comment:
Docs for `wss.clients`: https://github.com/powersync-ja/powersync-service/pull/36/files

The size of the set did seem to accurately reflect the number of concurrent connections in the testing I did.